src/squidpy/gr/_ligrec.py

```diff
@@ -735,11 +735,11 @@ def extractor(res: Sequence[TempResult]) -> TempResult:
     clustering = np.array(data["clusters"].values, dtype=np.int32)
 
     mean = groups.mean().values.T  # (n_genes, n_clusters)
-    mask = groups.apply(lambda c: ((c > 0).sum() / len(c)) >= threshold).values.T  # (n_genes, n_clusters)
+    mask = groups.apply(lambda c: ((c > 0).astype(int).sum() / len(c)) >= threshold).values.T  # (n_genes, n_clusters)
```

Comment:
Let's link to the issue on why this `astype` cast is there.  And use "int64" to be explicit (which is what `int` does anyway).